scripts/exgdas_enkf_earc.sh

```diff
@@ -48,7 +48,7 @@ if (( 10#${ENSGRP} > 0 )) && [[ ${HPSSARCH} = "YES" || ${LOCALARCH} = "YES" ]];
    SAVEWARMICB="NO"
    mm=$(echo "${CDATE}"|cut -c 5-6)
    dd=$(echo "${CDATE}"|cut -c 7-8)
-   nday=$(( (mm-1)*30+dd ))
+   nday=$(( (10#mm-1)*30+10#dd ))
```

Comment:
Are you sure this works? My attempts in the past show that once you start defining a base, you have to use a `$` even inside a math expression:
```
Orion-login-4 test/>x='09'
Orion-login-4 test/>a=$(( x + 1 ))
-bash: 09: value too great for base (error token is "09")
Orion-login-4 test/>a=$(( 10#x + 1 ))
-bash: 10#x: value too great for base (error token is "10#x")
Orion-login-4 test/>a=$(( 10#${x} + 1 ))
Orion-login-4 test/>echo $a
10
```

Reply:
@WalterKolczynski-NOAA No, it doesn't work.  I tested this with `LOCALARCH=NO`, so I essentially didn't test it.  I'll fix it up and give it a real test.

Reply:
Fixed (and tested) by adding `$`s.